boxtree/tree.py

```diff
@@ -175,26 +172,10 @@ def __post_init__(self):
         lows = bcenters[:, 0] - 0.5*self.root_extent
         highs = lows + self.root_extent
         self.bounding_box = [lows, highs]
```

Comment:
Does this play nice with `Tree` (which already has a `bounding_box` attribute)? I guess `Tree` doesn't call this `__post_init__` so it should be fine now, but in #56 that's a dataclass as well, so not quite sure..

Better to make this a separate property?

Reply:
Hm, this seems to need some `actx` help too, since `bcenters[:, 0]` gives a non-contiguous array and `pyopencl` doesn't appreciate that (I'm guessing that's why the `.get()` is there)

Reply:
I agree that this should be done consistently across `TreeOfBoxes` and `Tree`.

Reply:
Went ahead and made it an explicit field of `TreeOfBoxes` in [7a8848a](https://github.com/inducer/boxtree/pull/71/commits/7a8848a3bbdc098f99a1398a66b8eb09614c05d5).

Does that look ok? The only issue I can think of is that it's no longer recomputed from `box_centers` on a `replace`.